libs/langchain-community/src/chat_models/bedrock/web.ts

```diff
@@ -154,100 +154,310 @@ export interface BedrockChatFields
     BaseChatModelParams {}
 
 /**
- * A type of Large Language Model (LLM) that interacts with the Bedrock
- * service. It extends the base `LLM` class and implements the
- * `BaseBedrockInput` interface. The class is designed to authenticate and
- * interact with the Bedrock service, which is a part of Amazon Web
- * Services (AWS). It uses AWS credentials for authentication and can be
- * configured with various parameters such as the model to use, the AWS
- * region, and the maximum number of tokens to generate.
- *
- * The `BedrockChat` class supports both synchronous and asynchronous interactions with the model,
- * allowing for streaming responses and handling new token callbacks. It can be configured with
- * optional parameters like temperature, stop sequences, and guardrail settings for enhanced control
- * over the generated responses.
- *
- * @example
+ * AWS Bedrock chat model integration.
+ *
+ * Setup:
+ * Install `@langchain/community` and set the following environment variables:
+ *
+ * ```bash
+ * npm install @langchain/openai
+ * export BEDROCK_AWS_REGION="your-aws-region"
+ * export BEDROCK_AWS_SECRET_ACCESS_KEY="your-aws-secret-access-key"
+ * export BEDROCK_AWS_ACCESS_KEY_ID="your-aws-access-key-id"
+ * ```
+ *
+ * ## Key args
+ *
+ * ### [Init args](/classes/langchain_community_chat_models_bedrock.BedrockChat.html#constructor)
+ *
+ * ### [Runtime args](/interfaces/langchain_community_chat_models_bedrock_web.BedrockChatCallOptions.html)
+ *
+ * > See full list of supported init args and their descriptions in the [`constructor`](/classes/langchain_community_chat_models_bedrock.BedrockChat.html#constructor) section.
+ *
+ * ## Examples
```

Comment:
```suggestion
 * ```
 *
 * ## [Constructor args](/classes/langchain_community_chat_models_bedrock.BedrockChat.html#constructor)
 *
 * ## [Runtime args](/interfaces/langchain_community_chat_models_bedrock_web.BedrockChatCallOptions.html)
 *
 * Runtime args can be passed as the second argument to any of the base runnable methods `.invoke`. `.stream`, `.batch`, etc.
* They can also be passed via `.bind`, or the second arg in `.bindTools`, like shown in the examples below:
 *
 * ```typescript
 * // When calling `.bind`, call options should be passed via the first argument
 * const llmWithArgsBound = llm.bind({
 *   stop: ["\n"],
 *   tools: [...],
 * });
 *
 * // When calling `.bindTools`, call options should be passed via the second argument
 * const llmWithTools = llm.bindTools(
 *   [...],
 *   {
 *     stop: ["stop on this token!"],
 *   }
 * );
 * ```
 *
 * ## Examples
```